classes/Provider/Gateways/WebGatewayProvider.php

```diff
@@ -71,6 +73,18 @@ public function register(Container $app)
             return new Action\Page\TalkIdeasAction($app['twig']);
         };
 
+        $app[Action\Forgot\IndexAction::class] = function ($app) {
+            /** @var FormFactoryInterface $formFactory */
+            $formFactory = $app['form.factory'];
+
+            $resetForm = $formFactory->createBuilder(ForgotForm::class)->getForm();
```

Comment:
I believe we don't need to inject the entire `FormFactory` if all we need is an actual form.